temporalcli/commandsmd/commands.md

```diff
@@ -1,221 +1,537 @@
-        * `Options: <option>, <option>.` - Sets the possible options for a string enum type.
-        * `Env: <env-var>.` - Binds the environment variable to this flag.
-        * ``Alias: `--<alias>`.`` - Sets a flag as an alias of this one. The flag should not be separately defined.
-      * Options should be in order of most commonly used.
-    * Also can have single lines below options that say
-      `Includes options set for [<options-set-name>](#options-set-for-<options-set-link-name>).` which is the equivalent
-      of just having them appended to the bulleted list.
-      * Just because a command may have a couple of similar options with another doesn't mean you _have_ to make a
-        shareable options set. Copy/paste is acceptable.
-* Keep commands in alphabetical order.
-* Commands that have subcommands cannot be run on their own.
-* Keep lines at 120 chars if possible.
-* Add punctuation even at the end of phrases.
+This document is automatically parsed.
+Follow these rules.
+
+IN-HOUSE STYLE
+
+* Wording and grammar.
+  * Run a spell check.
+  * Be clear and concise.
+  * Don't reword the command in the short description. Use distinct supplementary language.
+    * Yes: temporal workflow delete: Remove Workflow Execution
+    * No: temporal workflow delete: Delete Workflow
+  * Re-use and adapt existing wording and phrases wherever possible.
+  * Word command short descriptions as if they began "This command will..."
+    Use sentence casing for the short description.
+  * ID is fully capitalized in text ("the Workflow ID") and Id in [metasyntax](https://en.wikipedia.org/wiki/Metasyntactic_variable) (YourWorkflowId).
+* Avoid parentheticals unless absolutely necessary.
+
+* Wrapping:
+  * Assume a user-visible line length of 80 characters max. `fmt -w 78` can help.
+    * Long descriptions must be pre-wrapped.
+  * When declaring Options follow the wrapping style used elsewhere in this file.
+    Splitting the items into multiple lines improves maintainability with clearer diffs.
+
+* Ordering:
+  * Commands: Use alphabetical order for commands.
+  * Options: Most commonly used options first.
+
+* Punctuation:
+  * With the exception of short descriptions and triple-quoted code-fenced samples, end everything with a period.
+    Do _not_ use periods for short descriptions.
+    Skipping required periods may cause errors in parsing this file.
+    Caution: Don't forget the period at the end of "Includes options set" lines.
+  * Introduce triple-quoted code-fenced samples with a colons.
+    Avoid using 'for example' unless there's no other way to introduce code.
+
+* Code, flags, and keys
+  * Demonstrate at least one example invocation of the command in every long description.
+  * Include the most commonly used patterns in long descriptions so users don't have to call help at multiple invocation levels.
+  * Avoid deprecated period-delineated versions of environment-specific keys.
+    * Yes:
```

Comment:
Note that this assumes everyone's terminals use `\` for wrapping which isn't true. But I don't think it's a big problem.

Reply:
It's a good point and thank  you for the call-out.

I think it's reasonable to assume the Unix standard is familiar to users who don't use backslash will know how to adapt if they're our target audience.